openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -441,7 +441,7 @@ The following is a non-normative example of an `authorization_details` object wi
 
 <{{examples/authorization_details.json}}
 
-The following is a non-normative example of an `authorization_details` object with a `format`:
+The following is a non-normative example of an `authorization_details` object requesting a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

Comment:
I think this reads more naturally without the second 'an' (hopefully I'm not contradicting other people's suggestions here...):

```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```